frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/DraggableSidebarLink.styled.tsx

```diff
@@ -30,6 +30,7 @@ export const StyledSidebarLink = styled(SidebarLink)<{ isDragging: boolean }>`
   ${props =>
     props.isDragging &&
     css`
+      pointer-events: none;
```

Comment:
Needed for a weird issue where onClick was being called on certain dragEnd calls